Userland/Applications/FontEditor/MainWidget.cpp

```diff
@@ -781,10 +785,11 @@ void MainWidget::set_show_system_emoji(bool show)
     m_glyph_map_widget->set_show_system_emoji(show);
 }
 
-ErrorOr<void> MainWidget::open_file(DeprecatedString const& path)
+ErrorOr<void> MainWidget::open_file(String const& path, NonnullOwnPtr<Core::File> file)
```

Comment:
MappedFile's API is `map_from_file(NonnullOwnPtr<Core::File>, StringView path);`

As such, we don't actually need the String here, we just need `path.view()`, and this function could take a StringView for the path instead. It's only used to set the mmap name on serenity anyway. The implicit conversion from String const& to StringView makes me very nervous.